au4/src/au3wrap/internal/au3audiodevicesmanager.cpp

```diff
@@ -20,6 +21,89 @@ void Au3AudioDevicesManager::init()
     initInputChannels();
 }
 
+QStringList Au3AudioDevicesManager::getAudioOutputDevices() const
+{
+    QStringList outputDevices;
```

Comment:
please don't use Qt here (use muse::String...) 
we try not to use Qt where it can be omitted :) i.e. Qt is used mainly only for view